src/marketplace/ListingsView.js

```diff
@@ -54,7 +58,7 @@ class ListingsView extends React.Component<Props, State> {
 
 	//Gets all the sells and filters them based on what type we are currently looking at
 	getMarketSales = async ():Promise<void> => {
-		itemsForSale = [];
+		let itemsForSaleArray = [];
```

Comment:
Make it const please. You can still push to it (the thing that is constant is that you don't set it to a new array; the array itself can change)

Reply:
Done